extensions/spark/kyuubi-spark-authz/pom.xml

```diff
@@ -331,4 +331,38 @@
         <testOutputDirectory>target/scala-${scala.binary.version}/test-classes</testOutputDirectory>
     </build>
 
+    <profiles>
+        <profile>
+            <id>genpolicy</id>
+            <activation>
+                <property>
+                    <name>ranger.version</name>
```

Comment:
Why do we need this property?

Reply:
Here adds no property, it acts as the trigger condition to ensure generating policy file is aligned with the targeted ranger version.